core/src/autogluon/core/_setup_utils.py

```diff
@@ -26,9 +26,9 @@
     "networkx": ">=3.0,<4",  # Major version cap
     "tqdm": ">=4.38,<5",  # Major version cap
     "Pillow": ">=10.0.1,<11",  # Major version cap
-    "torch": ">=2.2,<2.3",  # Major version cap, sync with common/src/autogluon/common/utils/try_import.py
-    "lightning": ">=2.1,<2.2",  # Major version cap
-    "pytorch_lightning": ">=2.2,<2.3",  # Major version cap, capping `lightning` does not cap `pytorch_lightning`!
+    "torch": ">=2.3,<2.4",  # Major version cap, sync with common/src/autogluon/common/utils/try_import.py
```

Comment:
Why are we pinning torch to >=2.3? This seems much too strict.

Reply:
I checked the previous releases and we have been restricting `torch` to be within one major version. If you feel this is too strict, i can make ti to be >=2.2